code-challenges.test.js

```diff
@@ -13,230 +10,50 @@ const { describe } = require("yargs")
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
+// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
-// --------------------1) Create a function that takes an object as an argument and decides if the number inside it is evenly divisible by three or not.
-
-//PSEUDOCODE: Declare a function named evenlyDiv
-//         Input:An object that takes an argument/parameter
-//         Operator used to check if a number is divisble: % modulo           
-//         Expected Output:A Boolean that lets us know whether or not the number evenly is divisble by 3       
-//        
-   const evenlyDiv = {
-   evenlyDivbyThree: "3",
-   notEvenlyDivbyThree: "19",
-   getData: function () {
-     return `${evenlyDiv['3'] % 3 == 0 }`
-    }
-   }  
-   console.log(evenlyDiv.getData())
-
- 
 // a) Create a test with expect statements for each of the variables provided.
-//PSEDUO CODE
-// Write a test for each of the statements
-// See a "good failure"
-//Use the "decribe()" convention
-//yarn add jest
-//yarn jest
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-describe("anItem", () => {
-   it("returns a number", () => {
-     expect(thisItem("yes")).toEqual("15 is divisible by three")
-   })
- })
- const object2 = { number: 0 }
- // Expected output: "0 is divisible by three"
- describe("thisOtherItem", () => {
-    it("returns a number", () => {
-      expect(object2("yes")).toEqual("0 is divisible by three")
-    })
-  })
- const object3 = { number: -7 }
- // Expected output: "-7 is not divisible by three"
- describe("lastItem", () => {
-    it("returns a number", () => {
-      expect(object1("yes")).toEqual("-7 is is not divisible by three")
-    })
-  })
-
-// I wrote a test for each of the statements above, with expect statements but each of them failed the test with the output in terminal (look below). Instead of getting "gppd failure" of ReferenceError: greeter is not defined, terminal told me that "object1" is not a function. 
-//orginally, inside of the describe statement I had "object 1" "object 2" and "object 3". But after the test failed, I changed it to "anItem" "thisOtherItem" and "lastItem" because I figured that it needed to be different from the expect statement. Both the describe and expect being the same is the reason I think the jest result told me that it was not a function. By the time I ran the test again, the Jest test suite failed to run.
```

Comment:
Your jest tests are written for the function you are making using the given variables, instead of for variables individually.